playbooks/openshift/openstack/cinder-registry.yml

```diff
@@ -19,6 +19,8 @@
       openshift_hosted_registry_storage_volume_size: "{{ disk.volume.size }}Gi"
       openshift_hosted_registry_storage_access_modes: "{{ openshift_hosted_registry_storage_access_modes | default(\"['ReadWriteOnce']\") }}"
       openshift_hosted_registry_storage_openstack_filesystem: "{{ openshift_hosted_registry_storage_openstack_filesystem | default('ext4')}}"
+      openshift_hosted_registry_storage_annotations:
+      - 'volume.beta.kubernetes.io/storage-class: standard'
```

Comment:
May want to have the `standard` set as a optional configuration - i.e.: something like:
```
- "'volume.beta.kubernetes.io/storage-class: {{ openshift_hosted_registry_storage_class | default('standard') }}'"
```

(note: I have not tested the above, so it may have syntax errors)

Reply:
Updated. Tested this and it works with removing the inner quotes,